### ISSUE TITLE
Entity and relation embedding when use_entity_embeddings=0

### ISSUE THREAD

**Issue by chizhik** [shehzaadzd/MINERVA#2]
Thanks for publishing the codebase. I have a question about pre-trained embeddings you mentioned in your paper. 
I was reading through your code and could not find the pre-trained embeddings. Do you use the result of Xavier initialization as embeddings when use_entity_embeddings flag is 0? It's curious that you achieve a high performance in this setting. Have you tried to compare xavier initialization with embeddings from TransE? What embeddings did you use for the results in your paper?

**Comment by shehzaadzd** [comment]
Hey, thanks for trying out our code! :)
We actually did not use pre-trained embeddings. We did experiment with TransE but found no particular improvement. When the use_entity_embedding flag is 0, the agent only chooses the next action/edge on the basis of the relation (and not the entity). We noticed that this worked better for certain datasets while using the entity embeddings worked better for certain other datasets.
